lib/php/lib/Factory/TStringFuncFactory.php

```diff
@@ -21,8 +21,9 @@
 
 namespace Thrift\Factory;
 
-use Thrift\StringFunc\Mbstring;
 use Thrift\StringFunc\Core;
+use Thrift\StringFunc\Mbstring;
```

Comment:
These changes seem unnecessary in order to change the line of documentation on line 53 below.

Reply:
I sorted use alphabetically, is it unnecessary? Am I expected that change it back?

Reply:
In general fewer changes are better but in this case no, I don't care.  I do the same cleanup when I'm in files sometimes. :)